gen_AbbrProps_reactGen.go

```diff
@@ -0,0 +1,68 @@
+// Code generated by reactGen. DO NOT EDIT.
```

Comment:
It's slightly annoying we now have a load of `gen_*Props_reactGen.go` files... but I'll fix up `reactGen` in a later PR so nothing to do in this PR; more of a note-to-self.